ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -0,0 +1,84 @@
+﻿@page
+@model ServerCore.Pages.Submissions.IndexModel
+
+@{
+    ViewData["Title"] = "Create";
+}
+
+<h2>Submissions</h2>
+<hr />
+<div>
+    <a asp-page="/Puzzles/Index" asp-route-eventid="@Model.EventId">Back to Puzzle List</a>
+</div>
+<br />
+<div class="row">
+    @if (!string.IsNullOrEmpty(Model.AnswerToken))
+    {
+        <div class="col-md-4">
+            <div class="alert alert-success" role="alert">
+                You got it! Correct answer: @Model.AnswerToken
+            </div>
+        </div>
+    }
+    else
+    {
+        <div class="col-md-4">
+            <form method="post">
+                <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+                <div class="form-group">
+                    <label asp-for="Submission.SubmissionText" class="control-label">Answer</label>
+                    <input asp-for="Submission.SubmissionText" class="form-control" />
+                    <span asp-validation-for="Submission.SubmissionText" class="text-danger"></span>
+                </div>
+                <div class="form-group">
+                    <input type="submit" value="Submit" class="btn btn-default" />
+                </div>
+            </form>
+        </div>
+    }
+
+</div>
+
+<div>
+    <table class="table">
+        <thead>
+            <tr>
+                <th>
+                    @Html.DisplayNameFor(model => model.Submissions[0].TimeSubmitted)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Submissions[0].SubmissionText)
+                </th>
+                <th>
+                    Response
+                </th>
+                <th></th>
+            </tr>
+        </thead>
+        <tbody>
+            @foreach (var item in Model.Submissions)
+            {
+                <tr>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.TimeSubmitted)
+                    </td>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.SubmissionText)
+                    </td>
+                    <td>
+                        @(item.Response != null ? item.Response.Note : "Incorrect")
```

Comment:
This isn't the response, at least historically it isn't. It's extra notes that the author can tag things with, for author/admin use only. If we are going to have this be the actual response let's name it better.


Note that I have never seen the Note field actually used in the old DB, so it's probably OK to repurpose this field - let's just pick a name that better matches the purpose.

Reply:
As we discussed in the meeting, please keep Note for authors to take notes on responses and use a separate field